sqlglot/expressions.py

```diff
@@ -759,6 +754,46 @@ class Property(Expression):
     arg_types = {"this": True, "value": True}
 
 
+class TableFormatProperty(Property):
```

Comment:
do you like having all these with the property name on it? seems a bit unnecessary since the subclass is already property

so just TableFormat and PartitionedBy etc

Reply:
Yeah I actually do like it. When someone is seeing these expressions referenced in code they don't see the class they inherited from. So it is nice making it explicit IMO. 

Also there is `AnonymousProperty` which really needs the property suffix to get it's purposes IMO. That also helps reinforce the point above.

Reply:
sounsd good